convert_test.go

```diff
@@ -162,6 +162,65 @@ func TestFromTime(base *testing.T) {
 	}
 }
 
+func TestFromTime_UTCVariants(t *testing.T) {
+	t.Parallel()
+	assert := testifyrequire.New(t)
+
+	// 1. Vanilla UTC
```

Comment:
Electing not to use a table driven test here (I don't expect to expand or contract these cases and the generation of `tz` is the most verbose part here).